sdk/storage/inc/blobs/internal/protocol/blob_rest_client.hpp

```diff
@@ -723,9 +724,9 @@ namespace Azure { namespace Storage { namespace Blobs {
     std::string ETag;
     std::string LastModified;
     uint64_t BlobContentLength = 0;
-    std::vector<std::pair<uint64_t, uint64_t>> PageRange;
-    std::vector<std::pair<uint64_t, uint64_t>> ClearRange;
-  }; // struct PageRangesInfo
+    std::vector<std::pair<uint64_t, uint64_t>> PageRanges;
```

Comment:
>uint64_t [](start = 26, length = 8)

`int64_t`?

Reply:
i'm going to change all these unsigned to singed in the future when `Nullable<T>` is available.